_sass/pages/_not_found.scss

```diff
@@ -1,11 +1,8 @@
 body.not-found {
   background-color: #112e51;
   color: #fff;
-  font-family: helvetica, helvetica neue, Ubuntu, segoe ui, arial, sans-serif;
-  height: 100%;
-  margin: 0;
   text-align: center;
-  text-shadow: 0 1px 3px rgba(0, 0, 0, 0.5);
+  display: grid;
```

Comment:
I was surprised to learn that Internet Explorer has [some support](https://developer.mozilla.org/en-US/docs/Web/CSS/display#browser_compatibility) for an old, non-standard CSS grid. But it doesn't appear to look correct, if we're trying to use this for vertically centering the content:

![image](https://user-images.githubusercontent.com/1779930/112045288-0d677e00-8b21-11eb-9ba2-8e6dd11aa5af.png)

Maybe flexbox would be a better option?

```suggestion
  display: flex;
  justify-content: center;
```

Reply:
This doesn't appear to be correct either. I might try using the original `table` and `table-cell` layout that I originally found to make this compatible.